### DISCUSSION TITLE
How to choose "elements" and "type_map"?

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/dpgen#388] [Q&A]
In a run_param.json file, "type_map": ["C","H","O"]. If "sys_configs" contains structures composed of different elements such as "C", "HO", and "CHO", can the corresponding POTCAR be generated? (does it mean that you need to train separately, then merge all the training sets, and finally complete the training?)

**Comment by AnguseZhang** [comment]
Don't worry about "composed of different elements such as "C", "HO", and "CHO" " . DP-GEN can automatically handle this  via `type_map` in dpdata, once you designate the correct correspondence bewteen  `type_map`  and `potcars` in `param.json`.
